### PR TITLE
Refactor unused var in presto-bigquery

### PR DESCRIPTION
**Pull request by brandboat**
#4188
This PR is a preparation before enable UnusedVariable error prone check.

### CODE REVIEW

File: presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ String selectSql(TableId table, String formattedColumns, String[] filters)
     {
         String tableName = fullTableName(table);
 
-        String whereClause = createWhereClause(filters)
+        String whereClause = createWhereClause()
```
Comment (Praveen2112) [inline presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java:164]: If `Optional.empty` always then can we remove whereClause ?

Comment (brandboat) [reply presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java:164]: OK, I will fix it up.

File: presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java [hunk 2]
```diff
@@ -32,11 +32,8 @@
 public class BigQueryConnectorModule
         implements Module
 {
-    private final NodeManager nodeManager;
-
-    public BigQueryConnectorModule(NodeManager nodeManager)
+    public BigQueryConnectorModule()
```
[the remark this answers is not in the document]
Comment (brandboat) [reply presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java:35]: Sure.

File: presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java [hunk 3]
```diff
@@ -160,12 +154,7 @@ String selectSql(TableId table, List<String> requiredColumns)
     String selectSql(TableId table, String formattedColumns, String[] filters)
```
Comment (ebyhr) [inline presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java:154]: `String[] filters` become unused var after this change. 

JFY: I'm looking into if we can revive `createWhereClause` method, but you don't need to care my work.

**Review by ebyhr — APPROVED** [review]

**Comment by wendigo** [comment]
@brandboat great idea to enable this check 👍🏻

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**